k2/csrc/array_ops.cu

```diff
@@ -147,63 +148,55 @@ Array1<int32_t> SpliceRowSplits(int32_t num_arrays,
 
 bool ValidateRowIds(Array1<int32_t> &row_ids, Array1<int32_t> *temp) {
```

Comment:
The passed `row_ids` should be `const`, i.e.,  `const Array1<int32_t> &row_ids`.

Reply:
Fixed